src/Ionide.ProjInfo/Library.fs

```diff
@@ -532,7 +532,7 @@ module ProjectLoader =
         let pi = project.CreateProjectInstance()
         getTfm pi isLegacyFrameworkProj
 
-    let createLoggers (paths: string seq) (binaryLogs: BinaryLogGeneration) (sw: StringWriter) =
+    let createLoggers (path: string) (binaryLogs: BinaryLogGeneration) (sw: StringWriter) =
```

Comment:
Since both call sites were only passing in a single path I changed this. Though it is public so would be a breaking change if someone was using it.